dataplicity/m2m/wsclient.py

```diff
@@ -388,6 +388,10 @@ def wait_ready(self, timeout=10):
         self.ready_event.wait(timeout)
         return self.identity
 
+    def terminate(self):
+        """Terminate the websocket."""
+        self.app.terminate()
```

Comment:
I can only see one call to `self.terminate()`, might changing that call to `self.app.terminate()` make sense instead of adding this method?